app/javascript/src/LoginPage/utils/formValidation.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import validate from 'validate.js'
+
+const constraintsTypes = {
```

Comment:
@hallelujah @damianpm 
To avoid duplication between client and server (regarding validation criteria) this should be generated by Ruby according to what's defined in the model.

Reply:
Yes, but it can be done as a following task: 
https://issues.jboss.org/browse/THREESCALE-3352
Is better to have 2 different tasks, this one is big enough to close it